Version 1.1.1 - cannot find module '@ricokahler/pool'

Upgrade from version 1.0.0 to version 1.1.1 is obviously a breaking change. 

Version 1.1.1 on NodeJS 14.15.4 produces the error
` Cannot find module '@ricokahler/pool'"`

Please use 2.x as new version or revert breaking change. 

Unfortunately I could not find updated documentation which describes migration path or how to upgrade.

--- a/package.json
+++ b/package.json
@@ -19,14 +19,18 @@
     "url": "https://github.com/ricokahler"
   },
   "sideEffects": false,
-  "exports": "./dist/index.modern.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js"
+    }
+  },
   "main": "./dist/index.js",
   "unpkg": "./dist/index.umd.js",
   "module": "./dist/index.module.js",
   "source": "./index.ts",
   "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "microbundle",
+    "build": "microbundle --target node",
     "prepare": "npm run build",
     "test": "jest"
   },

